src/server/database/heuristics.js

```diff
@@ -8,9 +8,21 @@
 */
 
 // --------- Fake Heuristics ---------------
-function run(layouts, callback) {
+function run(layouts, trimValues, callback) {
```

Comment:
trimValues shouldn't be needed.  Again, you should be passing in a number param to specify how many layouts to retrieve.  You shouldn't hard code ```10``` in this function.

Reply:
@d2fong  trimValues has been changed to numEntries which is passed on by the caller.